interface/patient_file/summary/stats_full.php

```diff
@@ -44,15 +46,11 @@
 <html>
 
 <head>
-<?php html_header_show();?>
 
-<link rel="stylesheet" href='<?php echo $css_header ?>' type='text/css'>
+    <?php Header::setupHeader(['dialog']); ?>
```

Comment:
dialog is automatically included when run setupHeader

Reply:
Well at least it's not escaping issue's! :)